content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -0,0 +1,111 @@
+# Generate live (real-time) metrics and plots for experiments
+
+When you
+[submit a new experiment](/doc/studio/user-guide/projects-and-experiments/run-experiments)
+from Iterative Studio, the
+[CI/CD setup](/doc/use-cases/ci-cd-for-machine-learning) in your Git repository
+gets invoked. If this setup includes a model training process, it will be
+triggered.
+
+In this model training CI action, you can use [DVCLive] to send live updates to
+metrics and plots back to Iterative Studio, without writing them to your Git
+repository. This will enable you to view all intermediate results in Studio
+while your experiment is still running.
+
+This requires a 3-step process:
+
+1. [Set up an access token](#set-up-an-access-token)
+2. [Configure your model training CI job](#configure-your-model-training-ci-job)
+3. [Send and view the updates](#send-the-updates)
+
+https://www.youtube.com/watch?v=hKf4twg832g
+
+TODO: Replace this with a video tutorial for live metrics and plots
+
+## Set up an access token
+
+Iterative Studio uses access tokens to authorize [DVCLive] to send live updates
+to the metrics and plots.
+
+### Generate access token
+
+To set up the access token, open your user profile page. In the
+`Studio access token` section, click on `Generate new token`.
+
+### Regenerate access token
+
+You can regenerate the access token at any point. The old access token will no
```

Comment:
A lot of structure here. Are these H3s necessary? There is one or 2 sentences under each one. ~~A numbered or bullet list would probably do, it may even be clearer.~~

Reply:
UPDATE: The whole section can be summarized into a paragraph TBH, maybe a single sentence. Something like (very crude):

> You can generate, regenerate, or delete your access tokens from the `Studio access token` section of your profile page.

Which makes me wonder if we even need the H2.